README.md

```diff
@@ -104,63 +111,79 @@ It will create a distributable native application.
 
-### 3. Specify plugin extensions
-
-Only if you need to, you can modify the default settings, by specifying
- the required extensions that the project uses in your `build.gradle` file:
-
-    gluonClient {
-        target = "host"
-        graalLibsVersion = "20.0.0-ea+12"
-        graalLibsPath = ""
-        javaStaticSdkVersion = "11-ea+6"
-        javafxStaticSdkVersion = "13-ea+7"
-        bundlesList = []
-        resourcesList = []
-        reflectionList = []
-        jniList = []
-        delayInitList = []
-        releaseSymbolsList = []
-        verbose = false
-        attachConfig {
-            version = "4.0.2"
-            services ''
-            configuration = "implementation"
-        }
-    }
-    
-### Requirements
+* Configure the runtime environment. Set `GRAALVM_HOME` environment variable to the GraalVM installation directory.
+
+For example:
+
+    export GRAALVM_HOME=/opt/graalvm-svm-darwin-20.1.0-ea+27
 
-At this moment the plugin is in beta, and supports Linux, Mac OS X and iOS platforms for now.
+* Set `JAVA_HOME` to point to the GraalVM installation directory
 
-To use the plugin to develop and deploy native applications on Mac or iOS platforms, you need a Mac with MacOS X 10.13.2 or superior, and Xcode 9.2 or superior, available from the Mac App Store. Once Xcode is downloaded and installed, open it and accept the license terms.
+For example:
 
-For now, only JDK 11 is supported. Any JDK 11 distribution that doesn't bundle JavaFX is valid, like:
+    export JAVA_HOME=$GRAALVM_HOME
 
-- [OpenJDK 11.0.2](https://download.java.net/java/GA/jdk11/9/GPL/openjdk-11.0.2_osx-x64_bin.tar.gz)
+By default `target` is set to `host`. To deploy to iOS, set the target:
 
-- [AdoptOpenJDK 11.0.3](https://github.com/AdoptOpenJDK/openjdk11-binaries/releases/download/jdk-11.0.3%2B7/OpenJDK11U-jdk_x64_mac_hotspot_11.0.3_7.tar.gz)
+```
+gluonClient {
+     target = "ios"
+ }
+```
 
-Once downloaded and installed, set `JAVA_HOME` pointing to that JDK 11.
+#### Linux and Android
+
+* Download this version of Graal VM: https://download2.gluonhq.com/substrate/graalvm/graalvm-svm-linux-20.1.0-ea+27.zip and unpack it like you would any other JDK. (e.g. in `/opt`)
+
+* Configure the runtime environment. Set `GRAALVM_HOME` environment variable to the GraalVM installation directory.
+
+For example:
+
+    export GRAALVM_HOME=/opt/graalvm-svm-linux-20.1.0-ea+27
+
+* Set `JAVA_HOME` to point to the GraalVM installation directory
+
+For example:
+
+    export JAVA_HOME=$GRAALVM_HOME
+
+By default `target` is set to `host`. To deploy to Android, set the target:
+
+```
+gluonClient {
+     target = "android"
+ }
+```
 
 Check the [documentation](https://docs.gluonhq.com/client) for more details about the plugin and running the [gradle samples](https://github.com/gluonhq/client-samples/tree/master/Gradle).
```

Comment:
May be we should remove this part as there is no Gradle in docs.gluonhq.com/client.